shacl12-core/index.html

```diff
@@ -2740,7 +2740,7 @@ <h3>Linking to shapes graphs (sh:shapesGraph)</h3>
 			<section id="validation-definition">
 				<h3>Validation</h3>
 				<p>
-					<dfn data-lt="validating|validates|validated|validate">Validation</dfn> is a mapping from some input
+					<dfn data-lt="validation|validating|validates|validated|validate">Validation</dfn> is a mapping from some input
```

Comment:
I *think* ReSpec does the right thing with varying capitalization, but if not, we'll need a bunch more `data-lt` variants.